frontend/src/features/Mission/components/MissionForm/slice.ts

```diff
@@ -0,0 +1,108 @@
+import { createSlice, current } from '@reduxjs/toolkit'
+import { isEqual } from 'lodash/fp'
+
+import type { MissionMainFormValues } from '../../../SideWindow/MissionForm/types'
+import type { MissionWithActionsDraft } from '../../types'
+import type { PayloadAction } from '@reduxjs/toolkit'
+
+export interface MissionFormState {
+  // TODO For side window closure prevention and cross-form validation we don't need the entire forms values.
+  // But we do for the map interactions.
```

Comment:
Le `draft` sert aussi aux modifications inter-composants Formik : ex. le VesselSearch qui modifie les codes FAOs.
On précise aussi ça pour compléter le commentaire ?

Reply:
Bah comme par principe un peu tous nos nouveaux slices sont de toute manière utilisés par beaucoup de parties, je me suis dit que ça ne servait plus de commenter ça. Maintenant qu'on a typé une grosse partie de l'app, si tu t'amusait à le retirer ou le renommer tu verrais immédiatement tout ce qui l'utilise, ce n'est pas suffisant comme check ?.

Reply:
Remove the `TODO`